pkg/util/util.go

```diff
@@ -61,6 +62,28 @@ func GetLocalIP(hostName string) (string, error) {
 	return ipAddr.String(), nil
 }
 
+// GetNodeIP returns node ip
+func GetNodeIP(hostname string, interfaceName string) (string, error) {
+	var hostIP string
```

Comment:
i think it's better to use using `edged` as receiver, and the above block to get ip can also use this function.
```
if config.Modules.Edged.CustomInterfaceName != "" {
				klog.Infof("Getting IP address by custom interface: %s", config.Modules.Edged.CustomInterfaceName)
				ip, err := 
....
```

Reply:
If i understand you correct, you asking me to move this GetNodeIP from pkgutil to edged as receiver. I can do that, Ok.
However the code section in your comment refers to the `server.go` at the very beginning. At his time there is no edged component existend at all. So i would suggest leaving this as it is the best option here.

Reply:
got it, you are right.